ui/pages/Search/components/SavedSearch.jsx

```diff
@@ -78,8 +78,8 @@ const SavedSearches = ({ options, savedSearchesByGuid, selectedSearch, load, loa
       loading={loading}
       placeholder="Select a Saved Search"
       value={(selectedSearch || {}).savedSearchGuid}
-      onChange={val => onChange(savedSearchesByGuid[val].search)}
-      options={options}
+      onChange={val => (val ? onChange(savedSearchesByGuid[val].search) : onChange({}))}
```

Comment:
nitpicky, but its a bit neater to do `val => onChange(savedSearchesByGuid[val].search || {})`